hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -97,15 +88,19 @@ def combine_gvcfs(mts):
     def localize(mt):
         return mt._localize_entries('__entries', '__cols')
 
-    cols = None
-    for mt in mts:
-        if cols is None:
```

Comment:
What was this code supposed to be doing before? It looks like it was completely deleted.

Reply:
This code was needed before CastMatrixTableToTable kept the column information. We never use `cols`, so the code became dead.